src/tests/logic/calculate.spec.js

```diff
@@ -0,0 +1,58 @@
+import calculate from '../../logic/calculate';
```

Comment:
- [ ] We need to add more tests here to test all calculator operators: +, -, *, /, %. For example:
expect that `5 * 20` will result in 100